code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -64,3 +64,7 @@
 	var/hardcore_survival_score = 0
 	/// For agendered spessmen, which body type to use
 	var/body_type = MALE
+
+	/// How many "units of blood" we have on our hands
+	var/bloody_hands = 0
```

Comment:
If this is a variable you added, it could use a better name. This sounds like a boolean, not like what is described as.
Perhaps something like `blood_in_hands` would be a better fit.

Reply:
I simply moved this from /mob to /mob/living/carbon/human and documented it. It was not introduced by me.

Reply:
Any chance of doing a mass-replacement?

Reply:
I considered it but felt that it's abit too out of scope of this PR, it already has a lot of changes. But sure I can do that.